index.html

```diff
@@ -8,7 +8,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>LinuxChix India</title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
```

Comment:
IMHO, there is not much effect on click rate. Also the site is pretty much static at the moment. Also, because of what @Slayslot told and also because of this: https://medium.com/@julianbez/4-reasons-not-to-disable-zoom-in-your-responsive-design-ad582c16a1c

I would suggest to allow users to zoom in/out.

Thanks! 

If you have any issues here or feel we are wrong on something, please let us know. Thanks!